### DISCUSSION TITLE
More blend modes in the BlendMode op

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#607] [Feature Requests]
Would be great to get a few more in here.
Difference, Negation, Exclusion would be ace. Would be so nice to work with these in 3d space.

**Comment by pandrr** [comment]
the texture blendmodes are written in 2d shaders, 3d mesh rendering does not work like that.
webgl does only offer those.

we could do something like threejs:
https://threejs.org/examples/#webgl_materials_blending_custom

i think this is very hard to understand, but it would just be handy to be able to cycle through them to get instant visual feedback...

but yeah its on my list for a long time already, not sure when i get to it
